app/controllers/api/functions.php

```diff
@@ -82,6 +85,8 @@
             '$id' => $functionId,
             'execute' => $execute,
             'enabled' => $enabled,
+            'installationId' => $installationId,
+            'repositoryId' => $repositoryId,
```

Comment:
I added multiple comments to UPDATE endpoint. Make sure to apply all of those here as well.

Reply:
Not adding logic to link repo in createFunction for now as discussed.